docs/io.cozy.calendar.md

```diff
@@ -0,0 +1,54 @@
+[Table of contents](README.md#table-of-contents)
+    
+# Cozy Calendar doctype
+
+## `io.cozy.calendar`
+
+The `io.cozy.calendar` doctype is loosely based on the [iCalendar RFC](https://datatracker.ietf.org/doc/html/rfc5545), in that some of the attributes have been renamed for clarity. The attributes with a `?` are optional.
+
+- `start?` : {date} (example: `"1959-05-15"`)
+- `end?` : {date} (example: `"1959-05-15"`)
+- `label?` : {string}
+- `description?` : {string}
+- `location?` : {string}
+- `organiser?` : {string}
+- `status?` : {string} `CONFIRMED|CANCELLED`
+- `attendee?` : {object} Array of string with event attendees
+
+### Example
+```json
+{
+  "_id": "62e5d66d6e11d19992b7efce794263f0",
+  "start": "20170422T010000",
+  "end": "20170422T010000",
```

Comment:
The example doesn't match what is described before. Here, it is a datetime, but the schema says it is a date, and there are dashes in `1959-05-15`, but not in `20170422T010000`.

Reply:
Removed useless examples and normalized format here -> https://github.com/cozy/cozy-doctypes/commit/b5d47c756bc54d94408ffbe3ded2b8577354a7c3